Makefile.fossa

```diff
@@ -0,0 +1,32 @@
+REPO_PATH?=docker/buildx
+BUILD_ANALYZER?=docker/fossa-analyzer
+FOSSA_OPTS?=--option all-tags:true --option allow-unresolved:true --no-ansi
+
+# realize the node dependencies by using yarn;
+# this will put the dependencies in `ui/node_modules`
+# for use by fossa-cli for license scanning
+.PHONY: fossa-yarn
+fossa-yarn:
+	docker run --rm -i -v "$$(pwd):$$(pwd)" -w "$$(pwd)/ui" node:8-alpine yarn --production=true
+
+clean-fossa-yarn:
+	$(RM) -r ui/node_modules
+
+compile:
+	$(MAKE) vendor
```

Comment:
You don't need to run vendor, we already have a check for vendor, if it fails we won't merge the PR. And it is not needed for compiling.

Reply:
removed entire section